diff --git a/node_modules/expo-dev-menu/android/build.gradle b/node_modules/expo-dev-menu/android/build.gradle
index f8817f1..31b5c3f 100644
--- a/node_modules/expo-dev-menu/android/build.gradle
+++ b/node_modules/expo-dev-menu/android/build.gradle
@@ -48,6 +48,11 @@ afterEvaluate {
       }
     }
   }
+  // workaround for https://github.com/wix/Detox/issues/4220
+  tasks.matching { it.name == 'packageReleaseAssets' }.all {
+        dependsOn ':expo-dev-menu:copyAssets'
+        mustRunAfter ':expo-dev-menu:copyAssets'
+  }
 }
 
 android {
